packages/nimble-components/src/wafer-map/modules/matrix-renderer.ts

```diff
@@ -0,0 +1,17 @@
+import * as Comlink from 'comlink';
+import type { WaferMap } from '..';
+import { workerCode } from '../workers/render-worker';
+import type { RenderWorker } from '../../../build/generate-workers/dist/esm/render-worker';
```

Comment:
This relative path seems suspicious. Why can't it just reference the type from its `source` folder location?

Reply:
When I try to use the path from source I get this error
<img width="333" alt="import from source error" src="https://github.com/ni/nimble/assets/110180309/88bfa7ec-e50b-4d42-a428-1d1113046613">
didn't get deep into that but my guess is that the source typescript file is outside of the nimble-components/src directory

Reply:
Oh I see. Sorry, I didn't recognize that at first.